Utils/github_workflow_scripts/handle_external_pr.py

```diff
@@ -180,6 +192,20 @@ def is_requires_security_reviewer(pr_files: list[str]) -> bool:
     return False
 
 
+def is_tim_reviewer_needed(packs_in_pr: set[str]) -> bool:
+    for pack in packs_in_pr:
+        pack_metadata = get_pack_metadata(pack)
+        for pr_file in pack:
+            if "yml" in pr_file:
+                if "feed: true" in pr_file:
+                    return True
+        tags = pack_metadata.get("tags")
+        categories = pack_metadata.get("categories")
+        if "Threat Intelligence Management" in tags or "Data Enrichment & Threat Intelligence" in categories:
```

Comment:
Change "Threat Intelligence Management" tag and "Data Enrichment & Threat Intelligence" category to be constants